File: tests/cases/dict_comp.py
#!/usr/bin/env python3


def show():
    # Simple dict comprehension
    squares = {x: x * x for x in range(5)}
    print(len(squares))

    # Dict comprehension with condition
    evens = {x: x * 2 for x in range(10) if x % 2 == 0}
    print(len(evens))


if __name__ == "__main__":
    show()

File: tests/expected/dict_comp.jl
function show()
squares = Dict(x => x*x for x in 0:5 - 1)
println(join([length(squares)], " "));
evens = Dict(x => x*2 for x in 0:10 - 1 if x % 2 == 0 )
println(join([length(evens)], " "));
end

function main()
show();
end

main()
